src/Navigation/Skill/index.js

```diff
@@ -1,11 +1,138 @@
 import React from "react";
+import data from "../../resume.json";
+import "./index.css";
 function Skills(props, ref) {
   return (
-    <div>
-      <h2 ref={ref} className="skills">
-        Skills
-      </h2>
-    </div>
+    <>
+      <div ref={ref} className="skills">
+        <div className="skills_profile">
+          <div className="skills_profile_logo"></div>
+          <h1 className="skills_profile_heading">SKILLS & KNOWLEDGE</h1>
+          <h5 className="skills_profile_sub-heading">How Good I'm</h5>
+        </div>
+        <div className="skills_web-development">
+          <div className="skills_web-development-content">
+            <div className="skills_web-development_heading_logo"></div>
+            <div className="skills_web-development_heading">
+              Web development
+            </div>
+          </div>
+          <div className="skills_web-development_content--part1">
+            <div className="skills_web-development_column">
+              <div className="skills_web-development_logo1"></div>
+              <div className="skills_web-development_name">
+                {data.skills[0].name}
+              </div>
+              <div className="skills_web-development_level">
+                {data.skills[0].level}
+              </div>
+            </div>
+            <div className="skills_web-development_column">
+              <div className="skills_web-development_logo2"></div>
+              <div className="skills_web-development_name">
+                {data.skills[1].name}
+              </div>
+              <div className="skills_web-development_level">
+                {data.skills[1].level}
+              </div>
+            </div>
+            <div className="skills_web-development_column">
```

Comment:
you don't need to write JSX blocks for each entry in `data.skills` array, can use `Array.map` here instead, like:
```
 data.skills[1].map(({name, level, logo}) => {
    return <></>;
  })
```
you can use two maps actually, one for iterating over skills as well:
```
  data.skills.map(subSkills => {
    return subSkills.map(({ name, level, logo }) => {
      return <></>;
    });
  });
```

Reply:
Object de-structuring